test.js

```diff
@@ -138,7 +138,7 @@ test('data.message calls back with error', function (t) {
   server = util.makeServer(testData)
     .on('ready', function () {
       ghutils.ghget(xtend(auth), urlBase, {}, function (err, data) {
-        t.deepEqual(err, new Error('Error from GitHub: borked borked'))
+        t.is(err.message, 'Error from GitHub: borked borked')
```

Comment:
This is a bug in the tests.

Reply:
How so? Does `deepEqual` not compare Error objects?

Reply:
I thought it did, but apparently not.

![screenshot from 2018-06-22 17-20-48](https://user-images.githubusercontent.com/308049/41784657-aa3607f4-7640-11e8-9853-e486a627343c.png)

The `console.log` logs `Error from GitHub: borked borked ([{"foo":"bar"}])` but `t.deepEqual()` says it's ok.

Reply:
I see. I'm guessing that's because `message` isn't enumerable.